lib/utils/fs_windows.go

```diff
@@ -27,7 +27,10 @@ limitations under the License.
 // was causing flock.Flock.TryRLock to return either "access denied" or "The process cannot access
 // the file because it is being used by another process".
```

Comment:
This comment used to apply to the entire file, not is only applies to `getPlatformLockFilePath`. I'd move it down.

Reply:
Moved